dastardcommander/dc.py

```diff
@@ -974,6 +1005,12 @@ def _startAbaco(self):
         for k, v in list(self.abacoCheckBoxes.items()):
             if v.isChecked():
                 activate.append(k)
+        
+        # Read the invertedChan list. Normalize it by making the list contain only unique and sorted values.
```

Comment:
I'm tempted to say we should error if there are non-unique values as it may represent an error in creating the list.